src/App.js

```diff
@@ -32,15 +33,69 @@ function App() {
       job: "github",
     },
   };
+  const [apiData, setApiData] = useState(UserData);
 
   const ContactIcons = [faLocationDot, faTwitter, faLink, faBuilding];
 
+  const searchHandler = async (username) => {
+    try {
+      const resUser = await fetch(`https://api.github.com/users/${username}`);
+      if (!resUser.ok) throw new Error("Problem getting user data");
+      console.log(resUser);
+      const data = await resUser.json();
+
+      const {
+        avatar_url,
+        name,
+        login,
+        created_at,
+        bio,
+        public_repos,
+        followers,
+        following,
+        location,
+        twitter_username,
+        blog,
+        company,
+      } = data;
+
+      setApiData({
```

Comment:
So, here we are simply massaging the data sent back to us to look like the default `UserData` that holds default data that shows on first render (this approach is also not the best, we need to change it - you will find another comment on it, we simply did that so we could merge the UI PR it had been open for a while)

By massaging this data, we are doing extra work and writing more code. Best approach would be to pass the entire `data` object to `setApidata` then that will be destructured further down the component tree.

```suggestion
      setApiData({
      setApiData(data);
```